### DISCUSSION TITLE
Book covers

### DISCUSSION THREAD

**Discussion by ghost** [Acclorite/book-story#44] [Q&A]
Hello, I have a question about the ui. 
In your screenshots your books lined up nicely in a grid with small three in a row covers, and it looks great. But I can't for the life of me find the setting for changing the size of the book covers on the library tab. They are huge and ugly looking. What am I doing wrong?

**Comment by Acclorite** [comment]
Can you provide the screenshot? 

About changing covers, you can do that from book's details (click the cover of the book in Library) by holding down the cover image, then "Change the cover image".

**Comment by ghost** [comment]
Sure. Here's what my reading tab looks like 
![Screenshot_2024-08-12-19-34-28-11_3bc43dc46a578ae94c988eb47f351bc1](https://github.com/user-attachments/assets/f63f599a-0aca-44ee-83bc-288f81277698)
And here's what yours looks like
![1](https://github.com/user-attachments/assets/e686afbb-d830-4880-b729-53a0e2e311b4)

I want my books to look like they do in your screenshots, i.e tidy and small 🙂

**Reply by Acclorite** [reply]
**Probably your screen resolution is small or/and your screen is small/narrow.** 

There is minimum width for Library Item to be displayed, if your screen cannot fit 3 items a row even by 1 pixel, then it will go to 2 items a row. I plan on improving customization in Library a lot, with custom items per row settings. Until then, I am not sure what to suggest..

_P.s. I thought it was some bug, but that does not look ugly to me? I honestly would prefer this way xD_

**Reply by ghost** [reply]
Oh, that makes sense xD It's probably too tall and narrow😅
